src/lib/draw/Toolbox.svelte

```diff
@@ -46,6 +47,8 @@
   let splitRouteMode: SplitRouteMode;
   let streetViewMode: StreetViewMode;
 
+  const unsubscribe = currentMode.subscribe((value) => mode = value);
```

Comment:
Why don't we remove the local var `mode` on line 40 then and just use the store everywhere? We can more or less just use `$currentMode` in place of `mode`, but I haven't thought through every use of it yet